index.js

```diff
@@ -10,8 +10,13 @@ const OK = chalk.green('Downloaded:');
 const FROM = chalk.gray('from');
 
 function download(repo, logger) {
-    utils.downloadRepo(repo).then((files) => {
-        files.forEach(file => logger.info(OK, `${file} ${FROM} ${repo}`));
+    utils.downloadRepo(repo).then(({ files, dependencies }) => {
+        if (files && files.length) {
+            files.forEach(file => logger.info(OK, `${file} ${FROM} ${repo}`));
+        }
+        if (dependencies && dependencies.length) {
+            dependencies.forEach(dep => logger.info(OK, `${dep} ${FROM} ${repo}`));
```

Comment:
I like it, but mixed feelings lol. I feel like, unlike NPM, our file structure matters because we're actually going to be using these files within our project. They're not buried under a no-mans-land directory.

So, seeing something like this:

```
foobar
├── .gitignore
└─┬ vutran/new
  ├─┬ foo.js
  │ ├─┬ vutran/editorconfig
  │ │ ├── .editorconfig
  │ │ └── yarn.lock
  │ └── bar.js
  └── readme.md
```

Might be far more confusing post-download than seeing this:

```
Downloaded: readme.md from foobar
Downloaded: .gitignore from foobar
Downloaded: .editorconfig from vutran/editorconfig
Downloaded: yarn.lock from vutran/editorconfig
Downloaded: foo.js from vutran/new
Downloaded: bar.js from vutran/new
```

Would start to show a difference in real boilerplates:

```
Downloaded: .editorconfig from lukeed/preact-starter
Downloaded: .gitignore from lukeed/preact-starter
Downloaded: config/babel.js from lukeed/preact-starter
Downloaded: config/setup.js from lukeed/preact-starter
Downloaded: config/uglify.js from lukeed/preact-starter
Downloaded: config/webpack.js from lukeed/preact-starter
Downloaded: license from lukeed/preact-starter
Downloaded: package.json from lukeed/preact-starter
Downloaded: src/index.html from lukeed/preact-starter
Downloaded: src/index.js from lukeed/preact-starter
Downloaded: src/index.sass from lukeed/preact-starter
Downloaded: src/views/index.js from lukeed/preact-starter
Downloaded: src/views/pages/article.js from lukeed/preact-starter
```

Colors help here, haha. But you get a get a list of clear list of filepaths alongside their source.